src/main/java/org/mule/extensions/jms/internal/source/JmsListener.java

```diff
@@ -207,7 +217,7 @@ public void onStart(SourceCallback<Object, JmsAttributes> sourceCallback) throws
       String msg = format("An error occurred while creating the consumers for destination [%s]: %s",
                           destination, e.getMessage());
       LOGGER.error(msg, e);
-      sourceCallback.onSourceException(new JmsExtensionException(e, msg));
+      throw new JmsExtensionException(e, msg);
```

Comment:
there's an open issue for that and the connectivity thing is going to be reworked anyway